src/interactive/Dropdown.jsx

```diff
@@ -0,0 +1,114 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import cx from 'classnames';
+import bindAll from '../utils/bindAll';
+
+/**
+ * @module Dropdown
+ */
+class Dropdown extends React.Component {
```

Comment:
Let's make this a `PureComponent` Most SDS components _should_ be `PureComponents` vs `Components`

*General philosophy on Stateless/PureComponent/Component:*
Stateless component -- these are the component that don’t need life-cycle methods or state. These should can be used in rendering repeat element of parent component such as rendering the text list which just displays the information and doesn't have any actions to perform. Caveats, cannot contain stateful component children.

Pure component -- these are the items which have life-cycle and does a baked in shallow comparison of props in shouldComponentUpdate. Use this for components that have simple prop structures and you expect don’t need to be complex components.

Normal Components or Complex Components -- these are complex components or general components the compose many children components and are connected to redux state. These are usually the page level components and consists lot of children components which read it’s data from this complex wrapper.